azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultOperation.java

```diff
@@ -40,29 +45,32 @@ public KeyVaultOperation(KeyVaultClient keyVaultClient, String vaultUri) {
 
     public String[] list() {
         try {
-            rwLock.readLock().lock();
-            return Collections.list(propertyNamesHashMap.keys()).toArray(new String[propertyNamesHashMap.size()]);
+            this.rwLock.readLock()
+                    .lock();
```

Comment:
Any reason why these(several below) `lock()` or `unlock()` operation are separated with newlines instead of write in one line?

Reply:
why wrapper line here ?